### PR TITLE
Hide permissions if read permission only (fixes #123)

### LINKED ISSUE #123: We should consider hidding resources permissions when we only have the read permission
This is related to https://github.com/mozilla-services/kinto/pull/88

When we have the ``read`` permission to list existing buckets, we can ask for a bucket and in that case we can list its permissions. I think it is only something that somebody that can modify this permission should be able to see.

### PR DESCRIPTION
**Pull request by leplatrem**
Fixes #123

- [x] Add documentation.
- [x] Add end view tests.
- [x] Decide whether it should be empty or removed from response
- [x] Bump protocol version
- [x] Add a changelog entry.

### CODE REVIEW

**Comment by leplatrem** [comment]
@n1k0+@magopian r?

File: CHANGELOG.rst [hunk 1]
```diff
@@ -11,6 +11,8 @@ This document describes changes between each past release.
 - Fix error response consistency with safe creations if the ``create`` permission
   is granted (fixes #792). The server now returns a ``412`` instead of a ``403`` if
   the ``If-None-Match: *`` header is provided and the ``create`` permission is granted.
+- The ``permissions`` is now empty in the response if the user has not the permission
```
Comment (n1k0) [inline CHANGELOG.rst]: The `permissions` *object*

File: tests/core/resource/test_object_permissions.py [hunk 2]
```diff
@@ -77,8 +84,8 @@ def setUp(self):
         self.record_uri = '/articles/%s' % record_id
         self.permission.add_principal_to_ace(self.record_uri,
                                              'read',
-                                             'fxa:user')
-        self.resource.model.current_principal = 'basicauth:userid'
+                                             'account:readonly')
+        # self.resource.model.current_principal = 'basicauth:userid'
```
Comment (n1k0) [inline tests/core/resource/test_object_permissions.py]: nit: leftover?

File: docs/api/index.rst [hunk 3]
```diff
@@ -11,6 +11,13 @@ API
 Changelog
 ---------
 
+1.11 (unreleased)
+'''''''''''''''''
+
+- The ``permissions`` are now empty in the response if the user has not the
```
Comment (magopian) [inline docs/api/index.rst]: somehow this doesn't sound very good... maybe reword to "if the user does not have the permission to write"?

**Comment by magopian** [comment]
all good it seems, :shipit: !
